### ISSUE TITLE
Rafsi of broda/brode/brodi/brodo/brodu

### ISSUE THREAD

**Issue by Wuzzy2** [lojban/cll#445]
The CLL leaves a couple of questions open that I like to have clarified in the CLL:

- To which gismu does the 4-letter rafsi "brod" belong? "broda", "brode", "brodi", "brodo" or "brodu"?
- Is it even allowed to use the “brod” rafsi (due to possible ambiguity)?

**Comment by Wuzzy2** [comment]
I consider this to be an actual bug. I like to note the CLL also has a clear contradiction in the rafsi chapter 4.6:

> Every gismu has from two to five rafsi, each of a different form, but each such rafsi represents only one gismu.
> (…)
> Each gismu always has at least two rafsi forms; one is the gismu itself (used only at the end of a lujvo), and one is the gismu with out its final vowel (used only at the beginning or middle of a lujvo).

This is a contradiction. Let me summarize:

Claim 1: Every gismu has a 4-letter rafsi
Claim 2: Every such rafsi represents only one gismu
Claim 3: “broda”, “brode”, “brodi”, “brodo”, “brodu” are gismu

So this means that “broda”, “brode”, “brodi”, “brodo”, “brodu” must *all* have the 4-letter rafsi “brod”. But then this rafsi can't represent only one gismu at the same time. That's a logical contradiction.

------

Possible solutions:

* Explicitly assign the “brod” rafsi to one of the 5 gismu. This implies that 4 of the 5 gismu won't have a 4-letter rafsi
* Remove the “brod” rafsi entirely. This implies that none of these 5 gismu will have a 4-letter rafsi

**Comment by lagleki** [comment]
Which do you prefer

**Comment by Wuzzy2** [comment]
Personally, I prefer assigning “brod” to “broda”, leaving the other 4 gismu without a 4-letter rafsi.

However, I STRONGLY suggest to discuss this with other Lojbanists first, before doing any changes.

**Comment by Wuzzy2** [comment]
Here's a different solution that I just thought of:
Assign “brod” to “brodi”, so that “brodi” can be used in the middle of the word as well. This is slightly more balanced as well.

-----

Summary:

#### Current situation: “brod” rafsi is ambigious

Theoretical rafsi list:

* broda: -rod-, -brod-, -broda
* brode: -bo'e-, -brod-, -brode
* brodi: -brod-, -brodi
* brodo: -brod-, -brodo
* brodu: -brod-, -brodu

The “brod” rafsi is could stand for any of the 5 rafsi (see explanation above). That's kinda meh.

#### Solution 1: Remove the “brod” rafsi

Rafsi list:

* broda: -rod-, -broda
* brode: -bo'e-, -brode
* brodi: -brodi
* brodo: -brodo
* brodu: -brodu

Only broda and brode can be used in the middle of a lujvo.
Also, the “brod” rafsi is probably already used in the wild, so it is probably not a good idea to remove it.

#### Solution 2: Assign “brod” to “broda” only

Rafsi list:

* broda: -rod-, -brod-, -broda
* brode: -bo'e-, -brode
* brodi: -brodi
* brodo: -brodo
* brodu: -brodu

Only broda and brode can be used in the middle of a lujvo.
Considering that broda already has a short rafsi which allows to use this word in the middle of a lujvo, it doesn't seem that useful to give it another rafsi.

#### Solution 3: Assign “brod” to “brodi” only:

Rafsi list:

* broda: -rod-, -broda
* brode: -bo'e-, -brode
* brodi: -brod-, -brodi
* brodo: -brodo
* brodu: -brodu

broda, brode and brodi can be used in the middle of a lujvo.
This solution seems now the most balanced to me, and is now my new preferred solution.

**Comment by phma** [comment]
I agree with assigning "-brod-" to "brodi". I guess that "broda" is the most frequently used and "brodu" the least. I use "rodbo'e" (along with "comdaidza") for "thingamajig".
